x-pack/plugins/ml/public/application/jobs/new_job/utils/use_field_stats_trigger.tsx

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { ReactNode, useCallback, useContext } from 'react';
+import {
+  EuiButtonIcon,
+  EuiComboBoxOptionOption,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHighlight,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import { FieldIcon } from '@kbn/react-field';
+import { EVENT_RATE_FIELD_ID, Field } from '../../../../../common/types/fields';
+import { MLJobWizardFieldStatsFlyoutContext } from '../pages/components/pick_fields_step/components/field_stats_flyout/field_stats_flyout';
+import { getKbnFieldIconType } from '../../../../../common/util/get_field_icon_types';
+
+interface Option extends EuiComboBoxOptionOption<string> {
+  field: Field;
+}
+export const useFieldStatsTrigger = () => {
+  const { setIsFlyoutVisible, setFieldName } = useContext(MLJobWizardFieldStatsFlyoutContext);
+  const renderOption = useCallback(
+    (option: EuiComboBoxOptionOption, searchValue: string): ReactNode => {
+      const field = (option as Option).field;
+      return option.isGroupLabelOption || !field ? (
+        option.label
+      ) : (
+        <EuiFlexGroup gutterSize="xs" alignItems="center">
+          <EuiFlexItem grow={false}>
+            <EuiButtonIcon
```

Comment:
Can you add a tooltip round this icon, saying something like `Inspect field stats`?

Reply:
Updated here [`43c65b5` (#147322)](https://github.com/elastic/kibana/pull/147322/commits/43c65b5efc206c71af2dcc46afbffd8bcda1b5fa)